docs/reference/server-config.md

```diff
@@ -359,6 +361,30 @@ Example:
 	</tr>
 </table>
 
+## Hot configuration reload
+We don't support to hot reload everything at present. What we support for now is:
+- increase logging level by signal `SIGUSR1`
+- decrease logging level by signal `SIGUSR2`
+- hot reload logging level from the config file by signal `SIGHUP`
```

Comment:
nit: can we change all three lines above to read `by signaling` instead of `by signal`?